database/core_templates/schema_init.sql

```diff
@@ -1363,28 +1363,28 @@ CREATE TABLE public.review_response (
     time_updated timestamptz DEFAULT CURRENT_TIMESTAMP,
     time_submitted timestamptz,
     is_visible_to_applicant boolean DEFAULT FALSE,
-    is_latest_review_submission boolean DEFAULT FALSE,
+    is_latest_review boolean DEFAULT FALSE,
```

Comment:
I'd wait until after we merge this before updating core_templates.

Reply:
Maybe just remove then?

Reply:
Yeah I guess